MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
```

Comment:
- [ ] [REQUIRED] Good Job so far by implementing what the previous reviewer requested. But please customize your license by  replacing  **[YOUR NAME ]** with your name and own info to make it more professional :smile: